src/useCases/postFeedback.ts

```diff
@@ -0,0 +1,29 @@
+import config from "lib/config"
+import getEmailer from "lib/getEmailer"
+import PromiseResult from "types/PromiseResult"
+import logger from "utils/logger"
+
+const postFeedback = (feedback: string, currentUserEmail: string): PromiseResult<void> => {
+  const sendFeedbackTo = "kayleigh.derricutt@madetech.cjsm.net" // sorry Kayleigh :(
```

Comment:
😂

Reply:
let the history books know that I am sorry. just in case anyone wants to point the finger at me after I am gone